tests/System.Buffers.Experimental.Tests/BytesReader.cs

```diff
@@ -10,13 +10,14 @@ namespace System.Slices.Tests
 {
     public partial class ReadOnlyBytesTests
     {
-        [Fact(Skip = "System.InvalidProgramException : Common Language Runtime detected an invalid program.")]
+        [Fact]
         public void SingleSegmentBytesReader()
         {
             ReadOnlyBytes bytes = Create("AB CD#EF&&");
             var reader = new BytesReader(bytes);
 
             var ab = reader.ReadBytesUntil((byte)' ');
+            Assert.True(ab.HasValue);
```

Comment:
It isn't really necessary. I added it during my investigation. However, it is a correct test but mostly benign.